src/ipumspy/api/core.py

```diff
@@ -231,6 +231,10 @@ def download_extract(
         extract: Union[BaseExtract, int],
         collection: Optional[str] = None,
         download_dir: Optional[FilenameType] = None,
+        stata_command_file: Optional[bool] = False,
+        spss_command_file: Optional[bool] = False,
+        sas_command_file: Optional[bool] = False,
+        R_command_file: Optional[bool] = False,
```

Comment:
Should this name be `r_command_file` to be consistent with python styling? I can see arguments both ways.

Reply:
Makes sense - since SPSS and SAS flags are lower case, R should probably be too.